bib/internal/setup/setup.go

```diff
@@ -100,23 +102,43 @@ func Validate() error {
 	return nil
 }
 
-// ValidateHasContainerStorageMounted checks that the container storage
-// is mounted inside the container
-func ValidateHasContainerStorageMounted() error {
+var insideContainer = func() (bool, error) {
 	if err := exec.Command("systemd-detect-virt", "-c").Run(); err != nil {
 		if exitErr, ok := err.(*exec.ExitError); ok {
 			// not running in a container, just exit
 			if exitErr.ExitCode() == 1 {
-				return nil
+				return true, nil
 			}
 		}
+		return false, err
+	}
+	return false, nil
+}
+
+// ValidateHasContainerStorageMounted checks that the container storage
+// is mounted inside the container
+func ValidateHasContainerStorageMounted() error {
+	inside, err := insideContainer()
+	if err != nil {
+		return err
+	}
+	if inside {
```

Comment:
However, this check is actually correct I think. ;)

Reply:
Yeah, *cough* :) Fixed!